### PR TITLE
remove usage of IOUtil.close

### PR DESCRIPTION
**Pull request by ebarboni**
remove deprecation and use try with resources to get rid of this old code

### CODE REVIEW

**Review by mbien — APPROVED** [review]
looks good. 

I think it would be ok to use regular formatting for updated code sections if you want. This would make the formatting patch in future smaller.

File: nbm-maven-plugin/src/main/java/org/apache/netbeans/nbm/CreateWebstartAppMojo.java [hunk 1]
```diff
@@ -616,66 +616,44 @@ private File copyLauncher( File standaloneBuildDir, File builtInstallation )
...
+        }
+        else
+        {
+            instream = getClass().getClassLoader().getResourceAsStream( resourcePath );
+        }
+        
+        FileOutputStream outstream = new FileOutputStream( destinationFile ); 
+        
+        try ( Reader source = new BufferedReader( new InputStreamReader( instream, "UTF-8" ) ); Writer destination = new OutputStreamWriter( outstream, "UTF-8" ); )
         {
```
Comment (mbien) [inline nbm-maven-plugin/src/main/java/org/apache/netbeans/nbm/CreateWebstartAppMojo.java:650]:
```java
        try (Reader source = new BufferedReader(new InputStreamReader(instream, "UTF-8")); 
             Writer destination = new OutputStreamWriter(outstream, "UTF-8"))  {
```

**Comment by ebarboni** [comment]
I will do on two line. I think I will have nervous breakdown and autoformat all and do a no op push to have in native NetBeans format like nb-pakage.

**Comment by mbien** [comment]
NB can format code sections. You select something and format.

**Comment by ebarboni** [comment]
Yep but changing for new code will make the source a patchwork

**Pull request merged**